snapshots/faostat/2023-02-22/faostat_ef.zip.dvc

```diff
@@ -0,0 +1,31 @@
+meta:
+  namespace: faostat
+  short_name: faostat_ef
+  name: 'Land, Inputs and Sustainability: Fertilizers indicators - FAO (2022)'
+  description: 'The FAOSTAT domain Fertilizers Indicators provides information on
+    three rations: a) the ratio between the totals by nutrient of agricultural use
+    of chemical or mineral fertilizers, reported in the FAOSTAT domain �Inputs/Fertilizers
+    by Nutrient� for nitrogen (N), phosphorus (expressed as P2O5) and potassium (expressed
+    as K2O) and the area of cropland reported in the FAOSTAT domain �Inputs/Land Use�;
+    b) The ratio of fertilizers use and the annual population reported in the FAOSTAT
+    domain �Population and Employment/Population�; and c) The ratio of fertilizers
```

Comment:
Are these unknown characters supposed to be there?

Reply:
Hmmm probably not, I'll check what happens in the upcoming steps and how these descriptions compare to the old descriptions. There's already another issue for improving description formats.